test/solidity-ibc/IntegrationTest.t.sol

```diff
@@ -32,15 +35,14 @@ contract IntegrationTest is Test {
     bytes[] public merklePrefix = [bytes("ibc"), bytes("")];
     bytes[] public singleSuccessAck = [ICS20Lib.SUCCESSFUL_ACKNOWLEDGEMENT_JSON];
 
-    address public sender;
-    string public senderStr;
-    address public receiver;
-    string public receiverStr = "someReceiver";
+    address public defaultSender;
```

Comment:
I refactored the integration tests to reduce duplication and to make it easier to reason about which values where used in different scenarios.